library/src/main/java/com/github/pwittchen/reactivesensors/library/ReactiveSensors.java

```diff
@@ -92,11 +93,30 @@ public boolean hasSensor(int sensorType) {
     return observeSensor(sensorType, samplingPeriodInUs, null);
   }
 
+  /**
+   * Returns RxJava Observable, which allows to monitor hardware sensors as a stream of
+   * ReactiveSensorEvent object with defined sampling period
+   *
+   * @param samplingPeriodInUs sampling period in microseconds, you can use predefined values from
+   * SensorManager class with prefix SENSOR_DELAY
+   * @param sensorTypes sensor type from Sensor class from Android SDK
+   * @return RxJava Observable with ReactiveSensorEvent
+   */
+  public Flowable<ReactiveSensorEvent> observeSensors(final int samplingPeriodInUs,
```

Comment:
I'd rename this method to e.g. `observeManySensors` - `observeSensor` and `observeSensors` looks similar and it's easy to make a mistake while using such API.